### PR TITLE
Update NFT-Marketplace-Part-2.md

### PR DESCRIPTION
**Pull request by yanukadeneth99**
- Changed the '../hardhat' to './hardhat' because running graph init does not move you to the newly created folder
- Added 'cd hardhat' to the removing of git folder else it's the main git folder that''ll get removed
- Updated what the subgraph looks like in v0.0.4
- Added quick definition for genesis block
- Added permanent discord invite link for celo-devhelp

### CODE REVIEW

File: NFT-Marketplace-Part-2.md [hunk 1]
```diff
@@ -147,89 +149,91 @@ There are three main files we will be touching in this project, and should take
...
+specVersion: 0.0.4
 schema:
   file: ./schema.graphql
 dataSources:
   - kind: ethereum
     name: NFTMarketplace
     network: celo-alfajores
     source:
-      address: "0x9014DD98Cd14B26c76069356247cE6d762018220"
+      address: "0x627f152f97d431B844604B4421313CA979712006"
```
Comment (haardikk21) [inline NFT-Marketplace-Part-2.md:212]: Can you also make this change (and other code changes) in `celo-nft-marketplace/subgraph` of this repo? Thanks a lot!

**Review by haardikk21 — CHANGES_REQUESTED** [review]

File: NFT-Marketplace-Part-2.md (hunk 1 above)
Comment (haardikk21) [reply NFT-Marketplace-Part-2.md:212]: Ideally do this as part of this same PR so easier to review

**Comment by yanukadeneth99** [comment]
@haardikk21 Change done in this PR.

**Comment by yanukadeneth99** [comment]
@haardikk21 should be okay now.

File: NFT-Marketplace-Part-2.md [hunk 2]
```diff
@@ -373,7 +377,7 @@ export function handleListingUpdated(event: ListingUpdated): void {
 }
 ```
 
-Awesome! Now, let's do `handleListingCanceled`. We don't want to display canceled listings on the marketplace, so we can just delete the entity from the datastore entirely.
+Awesome! Now, let's do `handleListingCanceled`. We don't want to display canceled listings on the marketplace, so we can just delete the entity from the datastore entirely. Insert the following code into the function.
 
 ```typescript
 export function handleListingCanceled(event: ListingCanceled): void {
```
Comment (haardikk21) [inline NFT-Marketplace-Part-2.md:383]: `ListingCanceled` should be changed to using 2 l's here as well

File: NFT-Marketplace-Part-2.md [hunk 3]
```diff
@@ -147,89 +149,91 @@ There are three main files we will be touching in this project, and should take
...
-      address: "0x9014DD98Cd14B26c76069356247cE6d762018220"
+      address: "0x627f152f97d431B844604B4421313CA979712006"
       abi: NFTMarketplace
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.5
       language: wasm/assemblyscript
       entities:
-        - ListingCanceled
+        - ListingCancelled
```
Comment (haardikk21) [inline NFT-Marketplace-Part-2.md:167]: This is an American vs British spelling thing. I'm fine changing it, but this also requires a change in the smart contract in part 1.

**Review by haardikk21 — CHANGES_REQUESTED** [review]

File: NFT-Marketplace-Part-2.md (hunk 2 above)
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-2.md]: Sorry, it should be good now.

**Comment by yanukadeneth99** [comment]
Sorry for tagging you alot. Appreciate this with all the work you've got!

**Pull request merged**